Dockerfile.server

```diff
@@ -1,7 +1,7 @@
 FROM public.ecr.aws/docker/library/ruby:2.7.7
 
-# Throw errors if Gemfile has been modified since Gemfile.lock
-RUN bundle config --global frozen 1
+ARG USE_CRON=false
```

Comment:
This build arg is used to determine whether Klaxon server container should setup the `rake check:all` command on a cron timer. Since the default value is fault, we don't need to worry about setting this variable during ECS deployment